docs/docs/get-started/quickstart.md

```diff
@@ -13,7 +14,7 @@ Welcome to the [Kurtosis][homepage] quickstart! This guide takes ~5 minutes and
 
 Kurtosis is a platform for packaging and launching environments of containerized services and serve two types of users: the author of an environment definition, referred to as a [Kurtosis package][packages-reference], and the consumer or user of the environment definition. This quickstart is intended to put you in the shoes of the consumer - someone who needs a quick and easy way to get a production-like environment to develop and test changes quicker for your distributed system. A separate guide is [available here][how-to-set-up-postgres-guide] to introduce you to the world of Kurtosis package authoring and how one might define and build an environment with Kurtosis for themselves or for their team.
```

Comment:
You mention "quick" twice in the same sentence, changed one of them to be more descriptive and I believe to be more grammatically correct.
```suggestion
Kurtosis is a platform for packaging and launching environments of containerized services and serve two types of users: the author of an environment definition, referred to as a [Kurtosis package][packages-reference], and the consumer or user of the environment definition. This quickstart is intended to put you in the shoes of the consumer - someone who needs a quick and easy way to get a production-like environment to develop and test changes against for your distributed system. A separate guide is [available here][how-to-set-up-postgres-guide] to introduce you to the world of Kurtosis package authoring and how one might define and build an environment with Kurtosis for themselves or for their team.
```

Reply:
this file was actually unedited and just moved around in this PR - I plan to completely rewrite this quickstart in the second iteration. will commit this for the short-term gain, but expect a complete rewrite of this file soon